lighthouse-cli/test/smokehouse/test-definitions/offline-local/offline-expectations.js

```diff
@@ -179,24 +179,22 @@ module.exports = [
       finalUrl: 'http://localhost:10503/offline-ready.html?broken',
       audits: {
         'installable-manifest': {
-          // Only starting in m89 do we get 'warn-not-offline-capable'.
-          // This only happens when a populated `fetch` handler that doesn't provide a 200 response
```

Comment:
can you move this comment to above the finalUrl i guess?

    // This page's SW has a `fetch` handler that doesn't provide a 200 response